### PR TITLE
feat: add poe tasks to ease development

### PR DESCRIPTION
**Pull request by jorgebodega**

### CODE REVIEW

File: conftest.py [hunk 1]
```diff
@@ -3,67 +3,72 @@
...
 
 @pytest.fixture(scope="session", autouse=True)
-def initialise_db():
+def initialise_db(database: TestingDatabase):
     """
     Migrates the database once before running tests
     """
     database.create_database()
+    yield
+    database.drop_database()
```
Comment (mercuriete) [inline conftest.py]: We agreed this is not acceptable.
performance problems and concurrence problems will occur.

**Review by mercuriete — CHANGES_REQUESTED** [review]

File: conftest.py (hunk 1 above)
Comment (jorgebodega) [reply conftest.py]: The scope of the fixture is `session`, so this is only executed before and after **all** test suites

Comment (mercuriete) [reply conftest.py]: we don't want that either.
we don't want scripts being executed on production.

you could add that script on a different execution than test.
if the test fails is something wrong in your local setup.
I am not comfortable with dropping all tables when launching tests even when we launch tests on an empty database.

**Review by morenocantoj — APPROVED** [review]
senkiu

**Pull request merged**